lib/_http_server.js

```diff
@@ -373,13 +373,13 @@ function connectionListener(socket) {
 function updateOutgoingData(socket, state, delta) {
   state.outgoingData += delta;
   if (socket._paused &&
-      state.outgoingData < socket._writableState.highWaterMark) {
+      state.outgoingData < socket.writeHWM) {
```

Comment:
This isn't correct.

Reply:
interesting enough, our test suite passed.

Reply:
I'll get it updated.